### PR TITLE
{bio}[intel/2016a] MRtrix 0.3.14 (REVIEW)

### PR DESCRIPTION
**Pull request by boegel**
requires https://github.com/hpcugent/easybuild-easyblocks/pull/932

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2159.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/9129eeb3b75abfbb845d81a70ca96adc for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2529.golett.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/077421e2405f4a743aa37956a6855b02 for a full test report.

File: easybuild/easyconfigs/m/MRtrix/MRtrix-0.3.14-intel-2016a-Python-2.7.11.eb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+description = """MRtrix provides a set of tools to perform diffusion-weighted MR white-matter tractography in a manner
+ robust to crossing fibres, using constrained spherical deconvolution (CSD) and probabilistic streamlines."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'cstd': 'c++11'}
+
+source_urls = ['https://github.com/MRtrix3/mrtrix3/archive/']
+sources = ['%(version)s.tar.gz']
+
+#patches = ['MRtrix-%(version)s.patch']
```
Comment (wpoely86) [inline easybuild/easyconfigs/m/MRtrix/MRtrix-0.3.14-intel-2016a-Python-2.7.11.eb]: why keep?

Comment (boegel) [inline easybuild/easyconfigs/m/MRtrix/MRtrix-0.3.14-intel-2016a-Python-2.7.11.eb]: typo ;)

**Comment by wpoely86** [comment]
:+1:

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**